README.md

```diff
@@ -1,48 +1,70 @@
-Till tech test
+[![Build Status](https://travis-ci.org/bagolol/till_tech_test.png)](https://travis-ci.org/bagolol/till_tech_test)
+
+
+till tech test
```

Comment:
Make sure you capitalise etc. properly - this goes to an employer and immediately looks sloppy